gallery-so/gallery
# Kaito/gltf fix

### Summary of Changes

gltf media isn't appearing on web desktop screen sizes because the model-viewer component had a height of zero even though it had height: 100%. I set the aspect-ratio to 1 so that they are able to expand according to its width.

also removed the a tag wrapping nft previews that navigate to the detail page if the displayed media is interactive (iframe or gltf)




### Demo or Before/After Pics
Before
![CleanShot 2024-03-20 at 18 31 15@2x](https://github.com/gallery-so/gallery/assets/80802871/231e9e2a-ad84-4407-8a44-1535923a1daa)

After
https://github.com/gallery-so/gallery/assets/80802871/dc21394f-a9d0-4d99-802c-96d434989378


### Edge Cases



### Testing Steps

go to these example gltfs
https://gallery.so/post/2dvJBXvnKBLg7uP8R9mZfxBTqeq
https://gallery.so/post/2duzQqLZLHexRdIKXDneen1Tmb1 
and confirm they load as expected on all screen sizes

### Checklist

Please make sure to review and check all of the following:
